queries/query_builders.go

```diff
@@ -164,11 +164,13 @@ func buildUpdateQuery(q *Query) (*bytes.Buffer, []interface{}) {
 		cols[i] = strmangle.IdentQuote(q.dialect.LQ, q.dialect.RQ, cols[i])
 	}
 
-	buf.WriteString(fmt.Sprintf(
-		" SET (%s) = (%s)",
-		strings.Join(cols, ", "),
-		strmangle.Placeholders(q.dialect.IndexPlaceholders, len(cols), 1, 1)),
-	)
+	var setSlice []string
```

Comment:
Since we know the number of columns, using `make([]string, len(cols))` seems appropriate to needless allocations.

Reply:
Update according to comments.